sw/device/tests/sim_dv/i2c_device_tx_rx_test.c

```diff
@@ -71,20 +70,45 @@ static volatile const uint8_t kI2cByteCount = 0;
 static volatile bool tx_empty_irq_seen = false;
 static volatile bool cmd_complete_irq_seen = false;
 
+/**
+ * This constant indicates the number of interrupt requests.
+ */
+#define NUM_I2C_IRQS 6
```

Comment:
Please use a anonymous enum instead of a macro for consistency, i.e.:

```
enum {
   kFoo = 6,
};
```

Reply:
Thanks. I've fixed it now.